api/peer/transport.go

```diff
@@ -25,7 +25,7 @@ package peer
 // Subscriber listens to changes of a Peer over time.
 type Subscriber interface {
 	// The Peer Notifies the Subscriber when its status changes (e.g. connections status, pending requests)
-	NotifyStatusChanged(Identifier)
+	NotifyStatusChanged(Identifier) error
```

Comment:
Fair enough, I did this change without being intelligent about it (which is almost never smart), I should have understood the context a bit better. I don't like just blanket ignoring the calls, but for now a `_ = ...` and a TODO: log would be ok, and when we do https://github.com/yarpc/yarpc-go/issues/574 we can handle that